addons/api/addon/serializers/auth-method.js

```diff
@@ -4,7 +4,28 @@
  */
 
 import ApplicationSerializer from './application';
-import { TYPE_AUTH_METHOD_OIDC } from 'api/models/auth-method';
+import { TYPE_AUTH_METHOD_PASSWORD } from 'api/models/auth-method';
+
+const fieldsByType = {
+  oidc: [
+    'state',
+    'issuer',
+    'client_id',
+    'client_secret',
+    'client_secret_mac',
+    'max_age',
+    'api_url_prefix',
+    'callback_url',
+    'disable_discovered_config_validation',
+    'dry_run',
+    'claims_scopes',
+    'signing_algorithms',
+    'allowed_audiences',
+    'idp_ca_certs',
+    'account_claim_maps',
+  ],
+  ldap: [],
```

Comment:
I assume this will be populated when we work on the serializer ticket?

Reply:
@DhariniJeeva yes!